backend/settings.py

```diff
@@ -36,7 +36,7 @@
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+    #'django.middleware.csrf.CsrfViewMiddleware', #SHOULD PROLLY ENABLE LATER MUST DISCUSS
```

Comment:
we chillin (Matt and I discussed that this won't be needed since we will eventually use `session-based` auth